### ISSUE TITLE
chat: line breaks ignored in quotes

### ISSUE THREAD

**Issue by tylershuster** [urbit/urbit#4298]
As entered:
```
> I heard there was a secret chord
> that palfun pressed and it got PRed
> but you don't really care for wikis, do you?
```

As rendered:
<img width="854" alt="image" src="https://user-images.githubusercontent.com/3999320/104815880-5111d480-57cc-11eb-8ff7-f089e34ebad0.png">

**Comment by tylershuster** [comment]
damn it inserted a typo too

**Comment by matildepark** [comment]
This is fixed as of #4289. Just gonna close this one prematurely as it is therefore a duplicate of #4202.

<img width="450" alt="image" src="https://user-images.githubusercontent.com/20846414/105248384-519bc980-5b44-11eb-8082-f24827f3993f.png">
